apps/src/templates/teacherDashboard/shapes.jsx

```diff
@@ -7,7 +7,7 @@ export const sectionShape = PropTypes.shape({
   name: PropTypes.string.isRequired,
   // Though we validate valid login types here, the server actually owns the
   // canonical list, and passes us the list of valid login types.
-  loginType: PropTypes.oneOf(Object.keys(SectionLoginType)).isRequired,
+  loginType: PropTypes.oneOf(Object.keys(SectionLoginType)),
```

Comment:
Under the new flow we don't populate a default login type for new sections (because we require the teacher to pick one) and loosening this validation lets me use this shape everywhere, even on the new section forms.